### DISCUSSION TITLE
「2022年に流行る技術予想」の利用規約が404: Not Found

### DISCUSSION THREAD

**Discussion by TORIFUKUKaiou** [increments/qiita-discussions#123] [要望・アイデア・不具合]
こんにちは。
はじめまして。
いつもQiitaさんにはたいへんお世話になっております。
ありがとうございます。Thanks a lotです。

さて、さっそく本題です。

[2022年に流行る技術予想](https://qiita.com/official-events/5cb794f7cb9ac194ed70)
の左側にある
利用規約のリンクが https://qiita.com/official-event-terms/forecast2022 です。

「お探しのページは見つかりません。
404: Not Found」
と書いてあるページに飛びます。

意図された通りなのでしょうか。

![スクリーンショット 2022-01-28 1 05 15](https://user-images.githubusercontent.com/4215723/151396807-a31aafcb-335e-42cd-8104-26ea8321a73a.png)

「利用規約」という語感とあくまでも私が想像する「利用規約」に感覚的なズレがありましたので、discussionを書きました。
私は古い人間ですので、私の感覚が時代遅れの可能性が大いにあるとおもっております。
お手数おかけしますがご確認のほどよろしくお願いいたします :pray::bow::pray::bow::pray::bow:

**Comment by kskwtnk** [comment]
@TORIFUKUKaiou 
大変申し訳ありません、こちら運営のミスです
ご指摘いただきありがとうございます！現在は修正いたしました

## 経緯

- 参加にあたり、アンケートの回答や個人情報の提供などを依頼するイベントでは、規約への同意をお願いしています
- 今回は上記のような依頼のないイベントだったため、規約自体が存在しておりません
- 設定ミスによりリンクだけ表示されてしまっていたため404を返してしまいました

重ねてになりますが、お手数をおかけして申し訳ありません！

**Reply by TORIFUKUKaiou** [reply]
Thanks a lot!
